### ISSUE TITLE
PHP Version Support

### ISSUE THREAD

**Issue by nschiffelbein** [XeroAPI/XeroOAuth-PHP#54]
@ronanq  Is there any chance the OAuthSimple library is going to be updated to use the newer __construct constructor standard for PHP7?

**Comment by tomcastleman** [comment]
Fixed in #55 - waiting on merge by @ronanq. In the meantime you can add the following repository to your `composer.json` if you want PHP 7 support:

```
    "repositories":[
        {
            "type":"vcs",
            "url":"https://github.com/tomcastleman/XeroOAuth-PHP.git"
        }
    ],
```

**Comment by ronanq** [comment]
Thanks @tomcastleman - I've just merged that
